packages/calculator/src/main.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import Calculator from '@pie-framework/material-ui-calculator';
+import './style.css';
+
+class Main extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = { showCalcByState: false };
+    }
+
+    showCalculatorHandler() {
+        this.setState({ showCalcByState: true });
+    }
+
+    hidecalculatorHandler() {
+        this.setState({ showCalcByState: false });
+    }
+
+    dragHandler = () => {
+        this.dragElement(document.getElementById("calculator-div"));
+    }
+
+    dragElement = (elmnt) => {
+        var pos1 = 0, pos2 = 0, pos3 = 0, pos4 = 0;
+        if (document.getElementById(elmnt.id + "-header")) {
+            /* if present, the header is where you move the DIV from:*/
+            document.getElementById(elmnt.id + "-header").onmousedown = dragMouseDown;
+        } else {
+            /* otherwise, move the DIV from anywhere inside the DIV:*/
+            elmnt.onmousedown = dragMouseDown;
+        }
+
+        function dragMouseDown(e) {
+            e = e || window.event;
+            // get the mouse cursor position at startup:
+            pos3 = e.clientX;
+            pos4 = e.clientY;
+            document.onmouseup = closeDragElement;
+            // call a function whenever the cursor moves:
+            document.onmousemove = elementDrag;
+        }
+
+        function elementDrag(e) {
+            e = e || window.event;
+            // calculate the new cursor position:
+            pos1 = pos3 - e.clientX;
+            pos2 = pos4 - e.clientY;
+            pos3 = e.clientX;
+            pos4 = e.clientY;
+            // set the element's new position:
+            elmnt.style.top = (elmnt.offsetTop - pos2) + "px";
+            elmnt.style.left = (elmnt.offsetLeft - pos1) + "px";
+        }
+
+        function closeDragElement() {
+            /* stop moving when mouse button is released:*/
+            document.onmouseup = null;
+            document.onmousemove = null;
+        }
+    }
+
+    render() {
+        const returnHtml = this.state.showCalcByState ?
+            <div id="calculator-div" onMouseEnter={() => this.dragHandler()} style={{ border: "1px solid black" }}>
+                <div id="calculator-div-header">Click here to move</div>
+                <Calculator mode="basic" onEvaluateExpression={(expression, result) => { }} />
+            </div>
+            : null;
+        return (
+            <div>
+                <button onClick={() => this.showCalculatorHandler()}>Open Calculator</button>
```

Comment:
use `material-ui/IconButton` - the best Icon I could find was this: https://materialdesignicons.com/icon/calculator - grab the svg and turn it into a React Component.